pysmiles/cheminfo.py

```diff
@@ -0,0 +1,182 @@
+"""
+Some simple cheminformatics tools.
+"""
+from collections import defaultdict
+import numpy as np
+import networkx as nx
+from .smiles_helper import PTE, remove_explicit_hydrogens
+
+def tanimoto_similarity(binarr1, binarr2):
+    """
+    Compute Tanimoto similarity between two binary arrays.
+
+    Parameters
+    ----------
+    binarr1: np.array
+    binarr2: np.array
+
+    Returns
+    -------
+    float
+        the tanimoto similarity
+    """
+    intersection = np.sum(binarr1 & binarr2)
+    union = np.sum(binarr1) + np.sum(binarr2) - intersection
+    return intersection / union if union != 0 else 0.0
+
+def features_to_binary(feature_list, size=1024):
+    """
+    Converte a list of features to a binary array of
+    given `size`.
+
+    Parameters
+    ----------
+    feature_list: abc.iteratable[int]
+        a iteratable with integer numbers
+    size: int
+        size of the binary array; default=1024
+
+    Returns
+    -------
+    np.array
+    """
+    fp_array = np.zeros(size, dtype=int)
+    nonzero = np.array(feature_list, dtype=int) % size
+    fp_array[nonzero] = 1
```

Comment:
I don't quite see how this affects the length of the feature_list

Reply:
not sure I understand the question.

Reply:
The goals is to fold feature_list into a binary array of a fixed size, no? Maybe I'm just unclear what feature_list is